package.json

```diff
@@ -42,8 +42,8 @@
     "readline-sync": "^1.4.10",
     "rimraf": "^3.0.2",
     "ts-jest": "^26.2.0",
-    "ts-node": "^10.1.0",
-    "typescript": "^3.9.7"
+    "ts-node": "^10.9.1",
+    "typescript": "^4.0.3"
```

Comment:
Had to upgrade Typescript to work with latest version of Axios. TS 4.0.3 is what we're currently using in front end.